include/deal.II/lac/read_write_vector.h

```diff
@@ -737,7 +737,8 @@ namespace LinearAlgebra
     /**
      * Pointer to the array of local elements of this vector.
      */
-    std::unique_ptr<Number[], decltype(std::free) *> values;
+    std::unique_ptr<Number[], /*decltype(std::free) * */ void (*)(void *)>
```

Comment:
`std::free` is ambiguous since there exist a `__host__` and a `__device__` variant.